CityPuzzle/Main Game/QuestPage.xaml.cs

```diff
@@ -247,5 +247,5 @@ public int CountPages()
                 nowSize = CountPages();
             }
         }
-        }
+    }
```

Comment:
This whole page is completely reworked in pull request #43 so refrain from doing any changes on it as it will result in merge conflicts